### PR TITLE
Add question checker

### PR DESCRIPTION
**Pull request by andersonjoseph**
This verifies if the input is a question, if not, a warning will be returned.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ class ShouldICliCommand extends Command {
     let question = args.question;
     let answer;
 
-    question = (question.endsWith('?')) ? question : question.concat('?');
```
Comment (cedricium) [inline src/index.js:18]: Lets add this check back in - users may potentially be use to leaving the `?` out, so I don't want this to start giving them warnings.

File: src/utils/index.js [hunk 2]
```diff
@@ -51,3 +51,8 @@ exports.yesSayings = [
   'yeah',
   'yes',
 ];
+
+//check if the input contain letters and ends with '?' 
+exports.isAQuestion = (input) => {
+  return RegExp(/[A-z]/g).test(input) && input.endsWith('?');
+}
```
Comment (cedricium) [inline src/utils/index.js]:
```sh
  55:1   error  Expected space or tab after '//' in comment  spaced-comment
  55:55  error  Trailing spaces not allowed                  no-trailing-spaces
  58:2   error  Missing semicolon                            semi
```

Couple things here:
- add a space after the `//`
- remove the trailing spaces at the end of line 55
- add a semicolon `;` after the `}` on line 58

Just to be safe, once you've made all those changes, do `npm run lint` and make sure there's no errors.

**Review by cedricium — CHANGES_REQUESTED** [review]
Thanks for the PR @andersonjoseph! I like what have done, just a few changes and I'd be happy to merge this awesome feature 👍

File: src/index.js (hunk 1 above)
Comment (andersonjoseph) [reply src/index.js:18]: Well, the feature focuses on give warnings if users forgets the '?' or if they don't include any letter. If the warnings are a problem I think there's nothing more to do.

Comment (cedricium) [reply src/index.js:18]: @andersonjoseph I was thinking about it and I think you're right - putting a `?` should be required. Go ahead and fix the other stuff and I'll get this merged!

**Review by cedricium — APPROVED** [review]

**Comment by cedricium** [comment]
Merged, thanks @andersonjoseph! 🚀

**Pull request merged**